src/huggingface_hub/hf_api.py

```diff
@@ -7906,6 +7909,12 @@ def update_inference_endpoint(
                 Inference Endpoint running on the `text-generation-inference` (TGI) framework (see examples).
             secrets (`Dict[str, str]`, *optional*):
                 Secret values to inject in the container environment.
+
+            domain (`str`, *optional*):
+                The custom domain for the Inference Endpoint (e.g. `"api-inference.endpoints.huggingface.tech"`).
+            path (`str`, *optional*):
+                The custom path for the Inference Endpoint (e.g. `"/models/google-bert/bert-base-uncased"`).
```

Comment:
path is a prefix match and should end with a / (auto corrected but if they want to be able to reach /whatever/path
on the backend the path should be /whatever)

Reply:
yay! perfect! - I updated the doc string